### DISCUSSION TITLE
Ignore .git on home folder

### DISCUSSION THREAD

**Discussion by carragom** [Bash-it/bash-it#2196] [Q&A] [answered]
Hi,

I normally keep my dot files under git, things like `/home/<user>/.bashrc` so there is a `.git` repo directly at my home folder. Which means that  bash-it always shows the git information for that repo whenever $PWD is under my home. Is there any way to tell bash-it to ignore that git repo under my home ?

Thanks.

**Comment by davidpfarrell** [comment, marked as answer]
Greetings ! This does sound like it could be irritating. My suggestion involves looking for ways to use a folder other than `.git` and these come to mind:

#### [yadm - Yet Another Dotfiles Manager](https://yadm.io/):

Utilizes git under the hood but purpose built to help manage dotfiles

#### [Dotfiles: Best Way to Store in a Bare Git Repository](https://www.atlassian.com/git/tutorials/dotfiles)

This blog post describes a technique to use an alias for `git` that configures an alternate folder for storage.

It came out of this HN discussion:
* [Ask HN: What do you use to manage dotfiles?](https://news.ycombinator.com/item?id=11070797)

Another blog post expands on this idea to manage dotfiles accross your entire machine:
* [Dotfiles Management](https://mitxela.com/projects/dotfiles_management)

The Atlassian post uses `config` as the alias, but the "Dotfiles Management" post suggests `dotfiles` as the alias, which I quite like.


I'd probably go with YADM if it can fit your needs since its purpose-built, but all of these ideas change the git folder name which will help hide it from bash-it ... 

Hope that helps - Lemme know what you think and if you give one of these a try ... 

-DF

**Reply by carragom** [reply]
Thanks a lot, yadm seems like what I'm looking for.
